test/test_enclave_wrangler.py

```diff
@@ -0,0 +1,89 @@
+"""Tests
+
+TODO's
+ - 1. Test framework: Current implementation is ad-hoc for purposes of development.
+ - 2. Change from validate to apply, or do both
+"""
+import os
+import sys
+
+TEST_DIR = os.path.dirname(__file__)
+PROJECT_ROOT = os.path.join(TEST_DIR, '..')
+# todo: why is this necessary in this case and almost never otherwise?
+# https://stackoverflow.com/questions/33862963/python-cant-find-my-module
+sys.path.insert(0, PROJECT_ROOT)
```

Comment:
@Sigfried `sys.path.insert(0, PROJECT_ROOT)` is how I solved the "unable to import/find enclave_wrangler" issue

What i wrote on teams:
> As I mentioned, this is one of my banes with sharing code. if i were to write a python package that is pip installable, ive never had a problem there
> 
> but when sharing a repository and running code that's separated out like we have it, i've experienced all kinds of import issues before
> 
> Anyway, the fix in the line above is sys.path.insert(0, PROJECT_ROOT). but this shouldn't be necessary. when running python from the command line, it's supposed to check for modules in the present working directory